packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -57,6 +60,33 @@ export const HelpCenterContactPage: React.FC = () => {
 		} );
 	}, [ isLoading, renderChat.state, renderEmail.render ] );
 
+	const liveChatHeaderText = useMemo( () => {
+		if ( isDefaultLocale( locale ) ) {
+			return __( 'Live chat', __i18n_text_domain__ );
+		}
+
+		return __( 'Live chat (English)', __i18n_text_domain__ );
+	}, [ __, locale ] );
+
+	const emailHeaderText = useMemo( () => {
+		if ( isDefaultLocale( locale ) ) {
+			return __( 'Email', __i18n_text_domain__ );
+		}
+
+		const isLanguageSupported = ( config( 'upwork_support_locales' ) as Array< string > ).includes(
+			locale
+		);
+
+		if ( isLanguageSupported ) {
+			const language = getLanguage( locale );
+			return language
+				? `${ __( 'Email', __i18n_text_domain__ ) } (${ language.name })`
```

Comment:
I think it'd be better to use `sprintf` and a string with placeholder, instead of template literal. For example:
```suggestion
			sprintf(
				/* translators: %s is a language name */
				__( 'Email (%s)' ),
				language.name
			)